CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 - Add `--show-ip-addresses` flag to `server list` command to optionally include IP addresses in command output.
-- Add `database connection list`, and `database connection cancel` commands.
+- Add `database connection list`, `database connection cancel`, `database start`, and `database stop` commands.
```

Comment:
personally I would keep each command as a separate item on the list, for me that easier to quickly parse with eyes. Not a blocker or anything ofc.